<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PickZ</class>
 <widget class="QWidget" name="PickZ">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>111</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="Z1Active">
        <property name="text">
         <string>Z1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="Z2Active">
        <property name="text">
         <string>Z2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="Z3Active">
        <property name="text">
         <string>Z3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="Z4Active">
        <property name="text">
         <string>Z4</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>237</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_ensure">
        <property name="text">
         <string>Ensure</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
